esperanto/malkovro/fajro.md

```diff
@@ -0,0 +1 @@
+Vi metas la fajron proksime de la muro , kaj ĝi hardas . Ĝi odoris [familiara](http://thesugardivorce.com/wp-content/uploads/2013/01/Perfect-Burned-Marshmallows-on-a-stick.jpg) , kiel odoro de via infanaĝo ...
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.